content/administration/upgrade/odoo_online.rst

```diff
@@ -1,30 +1,68 @@
-===========
-Odoo Online
-===========
+====================================
+Upgrade Odoo SaaS to a newer version
+====================================
 
-Requesting a test upgrade
-=========================
+In some cases, it is necessary to upgrade an Odoo SaaS database to a newer version. Reasons for
+doing so include removing a bug that occurs in the current version, gaining access to a feature that
+has been added in a version update, or simply because the company that owns the database wants to
+stay up to date with the newest version. Odoo databases can be manually upgraded directly from the
+main Odoo website.
 
-#. Go to your `database manager <https://www.odoo.com/my/databases/>`_
-#. Click on your profile icon and select *My Databases*.
+To upgrade an Odoo database, navigate to the `Databases <https://www.odoo.com/my/databases>`_ page
+and log in using the credentials of the account owner. The page will then display all of the Odoo
+databases associated with the user's account. Any databases that are not already on the most recent
+version of Odoo will display an :guilabel:`upgrade` button next to the name of the database.
 
-   .. image:: odoo_online/accessing-my-databases.png
-      :alt: Selecting My Databases under my profile
+.. image:: odoo_online/databases-page.png
+   :align: center
+   :alt: The My Databases page with an upgrade button next to the name of a database.
 
-#. Click on the up arrow button next to your main database name to proceed to the test upgrade.
+Click on the :guilabel:`upgrade` button and an :guilabel:`Upgrade your database` pop-up will appear.
+In the :guilabel:`drop-down` at the top of the pop-up, select the version of Odoo that the platform
+will be upgraded to. In the :guilabel:`Email to notify` field, enter an email address that will
+receive email notifications about the database upgrade.
 
-   .. image:: odoo_online/upgrade-option.png
-      :alt: Selecting the action settings icon
+There is also a :guilabel:`Purpose` section on the pop-up that is used to specify the reason for the
+upgrade. However, at this stage of the process, the only selectable option is :guilabel:`Test`. This
+is because Odoo requires users to create a test copy of the updated database before converting the
+actual database. This allows users to confirm that their database configuration will work with the
+newer version before making the upgrade permanent.
```

Comment:
```suggestion
There is also a :guilabel:`Purpose` section on the pop-up used to specify the reason for the
upgrade. However, at this stage of the process, the only selectable option is :guilabel:`Test`.

This is because Odoo requires users to create a test copy of the updated database before converting
the actual database. This allows users to confirm their database configuration works with the newer
version before making the upgrade permanent.
```